test/babelish/test_php2csv.rb

```diff
@@ -60,15 +60,15 @@ def test_initialize
       :csv_filename => csv_filename,
       :headers => headers,
       :filenames => filenames
-      })
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.